packages/expo-analytics-amplitude/android/src/main/java/expo/modules/analytics/amplitude/AmplitudeModule.java

```diff
@@ -90,6 +91,63 @@ public void setGroup(final String groupType, final List<Object> groupNames, Prom
     mClient.setGroup(groupType, new JSONArray(groupNames));
   }
 
+  @ExpoMethod
+  public void setTrackingOptions(final Map<String, boolean> options, Promise promise) {
```

Comment:
```suggestion
  public void setTrackingOptions(final ReadableArguments options, Promise promise) {
```

[`ReadableArguments`](https://github.com/expo/expo/blob/master/packages/@unimodules/core/android/src/main/java/org/unimodules/core/arguments/ReadableArguments.java) is a unimodules class similar to RN's `ReadableMap` which allows calls like `options.getBoolean('disableAdid', false)` which properly returns a boolean, not a pointer to an object.